src/robos.py

```
@@ -134,56 +133,66 @@ def sense_act(self):
             except Exception as e:
                 self.logger.error(f"Erro na ação do robô {self.id}: {e}")
 
+    def duelo(self, inimigo_id, target_pos):
+        """
+        Lógica de duelo entre dois robôs.
+        O robô com maior força (F) vence o duelo.
+        """
+        inimigo = self.robots_info.get(inimigo_id)
+        if inimigo:
+            self.logger.info(f"Duelo iniciado entre {self.id} e {inimigo_id}!")
+            if self.F > inimigo['F']:
```

Ruff (0.11.9)

Yoda condition detected

Rewrite as `inimigo['F'] < self.F`

(SIM300)